src/sidebar/util/oauth-client.js

```diff
@@ -208,11 +206,19 @@ class OAuthClient {
    * @param {Object} data - Parameter dictionary
    */
   _formPost(url, data) {
```

Comment:
Does this method have more general usefulness outside of OAuth?

Reply:
We don't post form data anywhere else in the client at the moment. If we did, then it might be useful.